### PR TITLE
Folder structure updated, removed redundant files, move nav/crumbs to app.js

### PR DESCRIPTION
**Pull request by cotikor**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import React, { useEffect } from 'react';
-import { Route, withRouter, Router } from 'react-router-dom';
+import React, { useState } from 'react';
```
Comment (timh1203) [inline client/src/App.js]: If not too much trouble, awesome if you could remove useState! Helps me debug easier. 💚

File: client/src/components/navigation/Navcrumbs.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const styles = theme => ({
 const Navcrumbs = props => {
   return (
     <Grid className={props.classes.wrapper}>
-      <p data-testid="crumbTrail">Home > Refreshrs </p>
+      <p data-testid="crumbTrail"> Home > {props.page} </p>
```
Comment (timh1203) [inline client/src/components/navigation/Navcrumbs.js]: Yessss! Dynamic work!

File: client/src/App.js (hunk 1 above)
Comment (kelfro) [reply client/src/App.js]: I don't think it's possible to remove this because it is used in line 14.

Comment (timh1203) [reply client/src/App.js]: Oh okay, it popped up when I ran the app. No worries then.

**Pull request merged**